restore-dotnet/action.yml

```diff
@@ -38,10 +38,9 @@ runs:
 
     - name: .NET Restore packages
       shell: bash
-      run: |
-        for f in $(find . -name "*.sln"); do echo "restoring solution $f" && \
-          dotnet restore $f --locked-mode
-        done
+      env:
+        DOTNET_NUGET_SIGNATURE_VERIFICATION: false
+      run: dotnet restore --locked-mode
```

Comment:
Are you sure that all SLN are in the root pa to of the repository? If yes, all good then. Otherwise, I’d suggest to keep the loop.

Reply:
https://github.com/trakx/trading-fees
https://github.com/trakx/messari-api-client
https://github.com/trakx/ubiquity-api-client
https://github.com/trakx/yousign-api-client
https://github.com/trakx/copper-api-client

These are the repos that don't have the .SLN in the root.
They are not active. If we need to update any of them, moving the SLN is easy.